string[] emotions
string gender
int8 age
